### DISCUSSION TITLE
How can I recover from a Scheduled backup?

### DISCUSSION THREAD

**Discussion by winston0410** [cloudnative-pg/cloudnative-pg#6055] [Q&A] [answered]
[From the doc](https://cloudnative-pg.io/documentation/1.20/recovery/#:~:text=You%20can%20use%20the%20data,parallel%20support%2C%20if%20requested).), it described how can we recover from an object store and from a `Backup` resource. But how can I recovery from `ScheduledBackup`? If I cannot recover from `SchdeuledBackup`, what is the use of it?

**Comment by alexbarcelo** [comment, marked as answer]
`ScheduledBackup` creates `Backup` objects following the given schedule.

So, if you create a `ScheduledBackup`, you will see Backup objects appear in your kubernetes. Remember that you can set `immediate: True` to force the creation of a backup when creating the ScheduledBackup resource (and then it will follow the schedule for subsequent ones).
